HardwareReport/HardwareReport.js

```diff
@@ -1166,27 +1166,88 @@ function load_airspeed(log) {
 }
 
 // Load GPS
-let gps
 const max_num_gps = 2
-function load_gps() {
+let num_gps = 0
+function load_gps(log) {
+    let gps = []
+
+    // for (let i = 0; i < max_num_gps; i++) {
+    //     let index = String(i+1)
+    //     if (i == 0) {
+    //         index = ""
+    //     }
+    //     const type_name = "GPS_TYPE" + index
+    //     if (type_name in params) {
+    //         const type = params[type_name]
+    //         if (type != 0) {
+    //             const pos_names = get_param_name_vector3("GPS_POS" + (i+1) + "_")
+    //             gps[i] = { type: type,
+    //                        pos: get_param_array(params, pos_names),
+    //                        node_id: params["GPS_CAN_NODEID" + (i+1)] }
+    //         }
+    //     }
+    // }
```

Comment:
Need to not break the existing functionality. We pull the position offsets for the 3D plot.